app/src/main/java/org/jellyfin/androidtv/ui/playback/AudioNowPlayingActivity.java

```diff
@@ -101,6 +102,8 @@
     private Lazy<BackgroundService> backgroundService = inject(BackgroundService.class);
     private Lazy<MediaManager> mediaManager = inject(MediaManager.class);
 
+    private PopupMenu mPopupMenu;
```

Comment:
```suggestion
    private PopupMenu popupMenu;
```

We don't use those c-style prefixes for variables